include/onnxruntime/core/framework/execution_provider.h

```diff
@@ -121,6 +135,22 @@ class IExecutionProvider {
 
   void InsertAllocator(AllocatorPtr allocator);
 
+  /**
+  Given a list of fused_node, return create_state/compute/release_state func for each node.
+  */
+  virtual common::Status Compile(const std::vector<onnxruntime::Node*>& fused_node,
```

Comment:
The reason is for brainslice, we could only load 1 firmware on the chip, so if there are multiple subgraphs, we have to compile them into 1 binary otherwise we need switch firmware during run call.